libs/base/Data/Strings.idr

```diff
@@ -15,15 +15,12 @@ partial
 foldl1 : (a -> a -> a) -> List a -> a
 foldl1 f (x::xs) = foldl f x xs
 
--- This works quickly because when string-append builds the result, it allocates
+-- This works quickly because when string-concat builds the result, it allocates
 -- enough room in advance so there's only one allocation, rather than lots!
+%foreign
+    "scheme:string-concat"
 export
 fastAppend : List String -> String
```

Comment:
If we are going to fix the FFI name & the comment,
should we also fix the exported name? This is a
breaking change but it also means we won't have to
keep living with this confusing name.

Reply:
This is used in the compiler, too, so it would require a rebootstrap, i guess?

Maybe we could add `fastAppend` as an alias to `fastConcat` and then mark it as `%deprecated` once someone adds `%deprecated`.

Not sure what to do here.

Reply:
Sounds good.